src/GraphQl/Serializer/ItemNormalizer.php

```diff
@@ -125,6 +129,10 @@ protected function setAttributeValue($object, $attribute, $value, $format = null
             $attribute = 'id';
         }
 
+        if ('{}' === $value) {
```

Comment:
That's weird. Why do we need this?

Reply:
When we dump `dump($context['args']['input']);` in  the DeserializeStage we got this : 
```
array:4 [
  "id" => "/related_dummies/2"
  "symfony" => "laravel"
  "thirdLevel" => "/third_levels/1"
  "embeddedDummy" => "{}"
]
```

Reply:
This looks like a bug in pho-GraphQL. Could you report it upstream and explain why we need this temporary fix in a comment please?

Reply:
Input is user-defined, there shouldn't be a fix in the serializer. Maybe the test is bad. Which test is failing?

Reply:
https://github.com/api-platform/core/runs/4595101310?check_suite_focus=true
Seems to be this one

Reply:
This is strange, we need to dig it, it's probably not the right fix.